### DISCUSSION TITLE
JSdoc support?

### DISCUSSION THREAD

**Discussion by RenaudDenis** [Aleph-Alpha/ts-rs#360] [Q&A]
I have not seen this in the documentation, but I am just wondering if there is any support for generating JSDoc?
Is it planned?

**Comment by gustavo-shigueo** [comment]
Hello @RenaudDenis! There is currently no support for exporting types as JSDoc. TypeScript types can be exported with a JSDoc summary attached by having your Rust types documented with `///` or `/***/` (see #187, #229 and #342) but that'll always just generate a TypeScript type with documentation, not a /** @typedef */ declaration

As for future support plans, I don't think it is planned, as we never did anything to help generate this type of declaration
